lib/test/setup.js

```diff
@@ -140,6 +140,7 @@ function requestTestPage(webRootDir, scriptSrcs, callback) {
 <head>
 </head>
 <body>
+    <div id="app"></div>
```

Comment:
+1 this is the correct way to do that (I did it in #49 too - we'll just merge them together)